spec/unit/lib/cloud_controller/runner_spec.rb

```diff
@@ -24,9 +26,45 @@ module VCAP::CloudController
     end
 
     describe '#run!' do
+      let(:runner) { Runner.new(argv + ['-c', config_file.path]) }
+
       it 'creates a pidfile' do
         expect(VCAP::PidFile).to receive(:new).with('/tmp/cloud_controller.pid')
-        subject.run!
+        runner.run!
```

Comment:
What's the reason for defining this `runner` `let` and not using `subject` here? Should have the same result, or?

Reply:
Good point. I thought with let I could pull out the initialisation of the runner out of the test context, so that if I would test for something which happens in the constructor, the test would fail. Unfortunately that does not work. I changed it back to use subject.run! again.